### DISCUSSION TITLE
Map issues

### DISCUSSION THREAD

**Discussion by Sky173** [albar965/littlenavmap#843] [Q&A]
I just updated to 'Windows - LittleNavmap-win-2.6.17.zip' and I'm having issues with the maps. Is there a way to fix this? Thanks!
![1](https://user-images.githubusercontent.com/70987356/151455800-55d9c9e5-a3f8-4d9f-ba9d-2e266ac16828.JPG)

...and when I zoom in.

![2](https://user-images.githubusercontent.com/70987356/151455824-2ebdb652-6ffb-4d5f-8e1a-10147d3f6a63.JPG)

**Comment by albar965** [comment]
Sorry, known issue. See here in the Avsim forum: https://www.avsim.com/forums/topic/613903-map-display-issues-with-openstreetmap-and-other-maps/?tab=comments#comment-4709913

You have to disable hillshading in LNM to fix this.

Alex

**Comment by Sky173** [comment]
Wow, thanks for the fast response! That fixed it! Love what you're doing.

**Comment by albar965** [comment]
You're welcome. :upside_down_face: 
And thank you for the kind words.
Alex
